README.md

```diff
@@ -37,6 +37,95 @@ Collection of form data is often not taken seriously, you are providing informat
 <h2 align="center" style="font-weight:bold">Backend and Frontend Setup</h2>
 
 
+
+## Contribution Guidelines 🏗
+
+Are we missing any of your favorite features, which you think you can add to it❓ We invite you to contribute to this project and improve it further
+
+To start contributing, follow the below guidelines: 
+
+🌟. Star🌟 the project to bookmark and appreciate the work.
+
+**0.**  Take a look at the existing [issues](https://github.com/felixfaisal/formica/issues) or create your own issues. Wait for the Issue to be assigned to you after which you can start working on it.
```

Comment:
Remove `0.`  keep it inside `<em></em>

Reply:
Okk sure

Reply:
Else number should I keep or remove it

Reply:
Done with it 
Please check

Reply:
Keep it just remove 0 numbering and put that statement inside the em tag.

Reply:
Updated recently.
Please check